v3/src/components/graph/adornments/adornment-types.ts

```diff
@@ -0,0 +1,6 @@
+export const kGraphAdornmentsClass = "graph-adornments-grid"
+export const kGraphAdornmentsClassSelector = `.${kGraphAdornmentsClass}`
+
+export const kMovableLineClass = "movabe-line"
+export const kMovableLineType = "Movable Line"
+export const kMovableLinePrefix = "ADRN"
```

Comment:
Ultimately, movable line-specific constants should be in a movable line-specific file, e.g. `movable-line-constants.ts` or `movable-line-types.ts` so that the movable line code can be self-contained.